packaging/mingw-w64/configure-build-msys2-mingw-static.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+
+cmake \
+    -G "MinGW Makefiles" \
+    -D CMAKE_INSTALL_PREFIX="$PWD/dest/" \
+    -D WITHOUT_WIN32_CONFIG=ON  \
+    -D RDKAFKA_BUILD_EXAMPLES=ON \
+    -D RDKAFKA_BUILD_TESTS=ON \
+    -D WITH_LIBDL=OFF \
+    -D WITH_PLUGINS=OFF \
+    -D WITH_SASL=ON \
+    -D WITH_SSL=ON \
+    -D WITH_ZLIB=OFF \
```

Comment:
Is there a reason not to enable zlib?

Reply:
I was using the existing script as a starting point. Looking through the CMakeLists.txt file though, I noticed that I like the default for _most_ of the options that were being manually specified, so I just removed them from the script. Now building WITH_PLUGINS and WITH_ZLIB